webapp/src/Controller/Jury/RejudgingController.php

```diff
@@ -94,12 +95,28 @@ public function __construct(
      */
     public function indexAction(Request $request): Response
     {
+        $curContest = $this->dj->getCurrentContest();
         /** @var Rejudging[] $rejudgings */
         $rejudgings = $this->em->createQueryBuilder()
+            ->select('r')
+            ->from(Rejudging::class, 'r');
+        if ($curContest !== NULL) {
+            $rejudgings = $rejudgings->join(Submission::class, 's', Join::WITH, "s.rejudging = r.rejudgingid")
```

Comment:
Why not ->join(r.submission, s)?

Reply:
Because when the `Join::WITH` is removed the JOIN is done without a "ON" in the query. I'm not sure how to fix that as I agree this is kinda ugly.

Reply:
Also if you use r.submission instead of submisssion::class? Shouldn’t be the case

Reply:
I think this is now better.